microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -1,20 +1,17 @@
 ---
-title: Microsoft Edge Add-ons API Reference (Beta)
+title: Microsoft Edge Add-ons API Reference
 description: The Add-ons API Reference, for REST endpoints to automate publishing updates to add-ons that are submitted to the Microsoft Edge Add-ons website.
 author: MSEdgeTeam
```

Comment:
Full-text search the repo for "Microsoft Edge Add-ons API Reference (Beta)" and update any link text, in case other articles point to these and have (Beta) in the link text.

Reply:
done